tidy3d/web/api/mode.py

```diff
@@ -319,7 +320,20 @@ def create(
         folder = Folder.get(folder_name, create=True)
 
         mode_solver.validate_pre_upload()
-        mode_solver.simulation.validate_pre_upload(source_required=False)
+        if isinstance(mode_solver.simulation, Simulation):
+            mode_solver.simulation.validate_pre_upload(source_required=False)
+        elif isinstance(mode_solver.simulation, EMESimulation):
+            # TODO: support this web api in the future
+            raise SetupError(
+                "'EMESimulation' is not yet supported in the "
+                "remote mode solver web api. Please construct a 'Simulation' "
+                "with the same 'medium', 'structures', 'grid_spec', 'symmetry', etc. as the "
+                "'EMESimulation' in order to use the remote mode solver. "
```

Comment:
should we provide a method to do this simply? and then we can support it?

Reply:
I previously had a method `EMESimulation.to_fdtd_sim`, but we decided not to keep it because the `Simulation` was not physically equivalent to the `EMESimulation`. However, it does make sense to have something like this just for this temporary purpose of enabling remote mode solving, until it is natively supported. So I'll add `EMESimulation._to_fdtd_sim` as a private method, and suggest using it in this error message.

Reply:
ok. although I'd say it's probably ok to make it public (probably strange to suggest using a private method). but maybe with a disclaimer in the docstring?